### DISCUSSION TITLE
Times instead of dot

### DISCUSSION THREAD

**Discussion by porcaror** [typst/typst#3153] [General]
When working in physics, the multiplication sign between number is often writte as e.g. 1.2X10, using the times symbol insted of the dot symbol. Would it be possible to have the opportunity to choose between the two symbols in a future release?
Thank you very much.

**Comment by Enter-tainer** [comment]
maybe you are sending issue to the wrong repo

**Comment by laurmaedje** [comment]
Could you clarify what exactly you want to write and what output you want to achieve?

**Comment by dnlsch** [comment]
I think you can already choose between `dot` and `times`:

```
$ 1.23 dot 10^99 $
$ 1.23 times 10^99 $
```

![grafik](https://github.com/typst/typst/assets/118688572/262382e2-0005-48fb-89dd-ec70794ec70a)

The package [unify](https://github.com/typst/packages/tree/main/packages/preview/unify/0.4.3) may also help you.

**Reply by porcaror** [reply]
I'm sorry. In fact I was speaking about the package unity, where the multiplication sign is only a dot, and I wanted to suggest to use the time symbol.

**Reply by Enivex** [reply]
> I'm sorry. In fact I was speaking about the package unity, where the multiplication sign is only a dot, and I wanted to suggest to use the time symbol.

You should create a feature request on the repository for unify then
